src/fundus/publishers/ch/zwanzig_minuten.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from typing import List, Optional
+
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class ZwanzigMinutenParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = XPath("//div[@class='Article_elementLead__N3pGr']/p")
+        _subheadline_selector = XPath("//section[@class='Article_body__60Liu']//h2[contains(@class, 'crosshead')]")
+        _paragraph_selector = XPath("//div[@class='Article_elementTextblockarray__WNyan']/p")
```

Comment:
The list box at the beginning of [this](https://www.20min.ch/story/hamas-chefs-yahya-sinwar-tot-reaktionen-israel-gazastreifen-103203976) article isn't completely extracted.